src/utils/adapters.js

```diff
@@ -621,3 +621,16 @@ export function adaptVerificationResult(verificationStatus) {
     status: translateCodeToStatus(verificationStatus.statusCode),
   }
 }
+
+export function adaptTopAccounts(topAccounts, distribution) {
+  return topAccounts
+    .slice(0, 100)
+    .map((account, index) => {
+      return {
+        rank: index + 1,
+        account: account.account,
+        balance: formatAePrice(formatAettosToAe(account.balance)),
+        percentage: (formatAettosToAe(account.balance) * 100 / distribution).toFixed(4),
```

Comment:
Is this calculated on the circulating supply?
I see the dashboard component uses `totalTokenSupply` from `blockChainStats.js` while here it seems `distribution` is different.

Reply:
you are right, fixed